.github/workflows/release.yml

```diff
@@ -203,3 +203,41 @@ jobs:
           for f in $(find dist/ -name '*.tgz'); do
             gh release upload $tag $f
           done
+
+      - name: Add charts to branch
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          version=$(jq -r '.version' <<< "${{ steps.goreleaser.outputs.metadata }}")
+          branch_version=$(cut -d'.' -f1,2 <<< "$version")
```

Comment:
Why not use the full version number? On a given minor charts release branch, will charts for a later patch release overwrite existing ones?

Reply:
Makes sense, thanks.